public/main.css

```diff
@@ -333,6 +333,7 @@ h2 {
     rgba(0, 0, 0, 1),
```

Comment:
in the line above, for our case to work we need `to left`. Then firefox will be fine on that page. As it seems that firefox is the only browser correctly using the unprefixed css and chromiums are stuck with the prefixed ones.

Reply:
Done :+1: , I have made the changes.